libcaf_core/caf/inspector_access_type.hpp

```diff
@@ -56,7 +56,7 @@ constexpr auto inspect_access_type() {
   //        > inspector_access > inspect > trivial > defaults.
   if constexpr (is_allowed_unsafe_message_type_v<T>)
     return inspector_access_type::unsafe{};
-  else if constexpr (std::is_array<T>::value)
+  else if constexpr (std::is_array<T>::value || is_stl_tuple_type_v<T>)
```

Comment:
This is a breaking change. Builtin-inspects must have precedence over the default implementation.

Reply:
I've disabled the `bugprone-branch-clone` in the end.